example/HMI/app/views/main.handlebars

```diff
@@ -1,3 +1,99 @@
+{{#tmplit 'Columns' maxColumns=2}}
```

Comment:
I dont understand what maxColumns does here

Reply:
Same, I only see a reference to it here, seems like a leftover.

From my understanding, columns are actually being set via CSS grid, with tmplits tokens (CSS vars).

Reply:
Its not doing anything. if deleted it still works.